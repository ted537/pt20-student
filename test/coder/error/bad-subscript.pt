using output;
var ints [1]: like 1;
put(ints[2]);
putln;
