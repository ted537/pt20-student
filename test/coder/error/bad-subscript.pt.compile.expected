line 3: subscript [2] out of bounds [0 .. 0]
